challenges/functions.js

```diff
@@ -7,18 +7,37 @@
   * The consume function should return the invocation of cb, passing a and b into cb as arguments
 */
 
+// A callback function is a function passed into another function as an argument, which is then invoked inside the outer function to complete some kind of routine or action
+//A Higher-Order function is a function that receives a function as an argument or returns the function as output.
+
+function consume(a, b, cb) {
+  console.log(cb(a, b)); // a and b are arguments for the cb
```

Comment:
dont forget to make sure you are returning vs console.log'ing where you need to!

Reply:
Yes. My bad.  I need to return and then console log the function, etc.